tests/collections/collection.test.ts

```diff
@@ -254,7 +263,7 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
             });
         });
     });
-    describe('findOne, findMany & filter tests', () => {
+    describe.only('findOne, findMany & filter tests', () => {
```

Comment:
Please get rid of `.only()` here, it is very dangerous to use `.only()` in case you accidentally commit it to GitHub, because that means you skip most of the test suite. I would strongly recommend using Mocha's `-g` flag instead of `.only()` as described here: https://masteringjs.io/tutorials/mocha/run-single-test

Reply:
sorry about that.
forget to remove it!